Makefile

```diff
@@ -102,22 +102,42 @@ prepare-version: version.txt
 version.txt:
 	@echo "2.$$(git rev-list HEAD --count).$$(git rev-parse --short HEAD | perl -ne 'print hex $$_')" > $@
 
+install-deps:
+	pip install flit
+	pip install ".[test]"
 
-.PHONY: build-deb-package
-build-deb-package: prepare-changelog
-	@echo 'Building debian package'
+lint: black isort #pylint mypy bandit
 
-	rm -f pyproject.toml  # for forcing usage of setup.py
-	cd debian && \
-		debuild --check-dirname-level 0 --preserve-env --no-lintian --no-tgz-check -uc -us
+black: install-deps
+	black --check src/ tests/
```

Comment:
Let's pass `-S/--skip-string-normalization` option here to skip string normalization